tests/test_1d_arrays.py

```diff
@@ -104,20 +104,26 @@ def test_passing_0d_arrays(self):
     def test_passing_0d_arrays_1(self):
         dim = np.random.randint(5)
         # (np.zeros([0, dim]), np.zeros([dim, 0]), np.zeros([0]), np.zeros([0, 0])):
+        dim = 2
 
         a = np.zeros([0, dim])
         v = np.ones([10, 10])
-        f = np.ones([10, 10], dtype=np.int)
+        f = np.ones([10, 10], dtype=np.int64)
 
         retv, retp = npe_test.one_d_arg(v, f, a)
-        self.assertEqual(retp.shape, a.shape)
-        self.assertEqual(len(a), 0)
+        # Alec: I'm not sure if it's expected behavior in numpy or a bug in
+        # numpyegien, but if dim == 1 then retp.shape = (0,) rather than (0,1)
+        if dim == 1:
```

Comment:
NumpyEigen squeezes 1 dimensions by default when returning values, but there's an argument to `npe::move` (maybe it's hidden -- I don't recall atm) that changes this behavior.